compiler-utils/src/testFixtures/java/com/sqareup/anvil/compiler/internal/testing/TestUtils.kt

```diff
@@ -0,0 +1,181 @@
+@file:Suppress("unused")
+
+package com.sqareup.anvil.compiler.internal.testing
```

Comment:
Another sqareup.

Reply:
Oh damn, nice catch. I'll address this in a follow up PR. Stacking is annoying sometimes.

Reply:
https://github.com/square/anvil/pull/293